dora/stress/common/src/main/java/alluxio/stress/worker/WorkerBenchParameters.java

```diff
@@ -61,14 +61,6 @@ public final class WorkerBenchParameters extends FileSystemParameters {
       description = "The buffer size for IO operations. (1k, 16k, etc.)")
   public String mBufferSize = "4k";
 
-  @Parameter(names = {"--block-size"},
-      description = "The size of the file block. (16k, 64m, etc.)")
-  public String mBlockSize = "32m";
-
-  @Parameter(names = {"--num-files"},
```

Comment:
we now calculate this number from the concurrency params, i.e. num-workers and threads